extendr-api/src/robj/into_robj.rs

```diff
@@ -570,38 +740,28 @@ macro_rules! impl_from_as_iterator {
 //     }
 // }
 
-// TODO: Is this still relevant?
-// We would love to do a blanket IntoIterator impl.
-// But the matching rules would clash with the above.
-
-impl<'a, T, const N: usize> From<[T; N]> for Robj
-where
-    Self: 'a,
-    T: ToVectorValue,
-{
-    fn from(val: [T; N]) -> Self {
-        val.into_iter().collect_robj()
-    }
-}
-
-impl<T> From<Vec<T>> for Robj
-where
-    T: ToVectorValue,
-{
-    fn from(value: Vec<T>) -> Self {
-        value.into_iter().collect_robj()
-    }
-}
-
-impl<'a, T> From<&'a Vec<T>> for Robj
-where
-    T: 'a,
-    &'a T: into_robj::ToVectorValue,
-{
-    fn from(value: &'a Vec<T>) -> Self {
-        value.iter().collect_robj()
-    }
-}
+// Template constants are still unstable in rust.
+impl_from_iter! {[T; 1]}
+impl_from_iter! {[T; 2]}
+impl_from_iter! {[T; 3]}
+impl_from_iter! {[T; 4]}
+impl_from_iter! {[T; 5]}
+impl_from_iter! {[T; 6]}
+impl_from_iter! {[T; 7]}
+impl_from_iter! {[T; 8]}
+impl_from_iter! {[T; 9]}
+impl_from_iter! {[T; 10]}
+impl_from_iter! {[T; 11]}
+impl_from_iter! {[T; 12]}
+impl_from_iter! {[T; 13]}
+impl_from_iter! {[T; 14]}
+impl_from_iter! {[T; 15]}
+impl_from_iter! {[T; 16]}
+impl_from_iter! {[T; 17]}
+impl_from_iter! {[T; 18]}
+impl_from_iter! {[T; 19]}
+impl_from_iter! {Vec<T>}
+impl_from_iter! {&Vec<T>}
```

Comment:
How is this relevant to error handling?

Reply:
I think the diff viewer is off here. I will try to add a comment to hack it back into order

Reply:
fixed https://github.com/extendr/extendr/pull/473/commits/9923d2464eb2d32636826d1d51de32e3b856ccd7